pimd/pim6_cmd.c

```diff
@@ -1856,20 +1856,21 @@ DEFPY (show_ipv6_pim_secondary,
 	return pim_show_secondary_helper(vrf, vty);
 }
 
-DEFPY (show_ipv6_pim_cand_rp,
```

Comment:
Where did this command go?  Where is the backwards compatiblity for this?  Someone is using this command we cannot just remove it.

Reply:
This is a new command. I added new show command beyond what we have in the first couple of commits in a later commit. I didn't go back to make the change in the original commit to avoid some code churn, but I can do that if you think it is necessary.